src/main/java/utils/Splitter.java

```diff
@@ -0,0 +1,24 @@
+package utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Splitter {
+    private static final String DEFAULT_DELIMITERS = ",|:";
+    private static final String CUSTOM_DELIMITER_REGEX = "^//(.)\n(.*)";
+    private static final Pattern pattern = Pattern.compile(CUSTOM_DELIMITER_REGEX);
+    private static Matcher matcher;
+
```

Comment:
static 메소드만 존재하는 클래스의 경우 객체 생성을 막기위해 
```suggestion
private Splitter() {}
```
와 같이 막아두는 방법도 있습니다 : )

Reply:
의도한 대로 사용하게 하려면 막아두는 것이 좋겠네요~! 👍